presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java

```diff
@@ -39,19 +41,21 @@
 {
     private final ServerInfo serverInfo;
     private final GracefulShutdownHandler shutdownHandler;
+    private final long startTime;
 
     @Inject
     public ServerInfoResource(ServerInfo serverInfo, GracefulShutdownHandler shutdownHandler)
     {
         this.serverInfo = requireNonNull(serverInfo, "serverInfo is null");
         this.shutdownHandler = requireNonNull(shutdownHandler, "shutdownHandler is null");
+        this.startTime = System.nanoTime();
     }
 
     @GET
     @Produces(APPLICATION_JSON)
-    public ServerInfo getServerInfo()
+    public CurrentServerInfo getServerInfo()
```

Comment:
This is a breaking change and doesn't seem necessary. Why not add uptime to `ServerInfo`?